docs/source/quick_start/troubleshooting.rst

```diff
@@ -0,0 +1,85 @@
+
+Troubleshooting
+===============
+
+Have problems to make things work? Here are some common issues and their solutions
+
+MCDR won't start
+----------------
+
+* **mcdreforged: command not found**
+    * Try ``python -m mcdreforged``
+    * If you installed MCDR via pipx, ensure the pipx apps directory is in your PATH
+    * If you installed MCDR via pip globally, ensure the pip scripts directory is in your PATH
+    * If you installed MCDR via pip within venv, make sure you have activated the venv
+    * If you installed MCDR via system package manager... don't install MCDR via system package manager
+
+* **There're some error raised by Python**
```

Comment:
It could also be MCDR itself being buggy. So concating the community / submitting an issue with logs is also an option here

Reply:
in the end of the part there is:
```rst
Get in touch with :ref:`our kind community <quick_start/next_steps:Community>`
```
which is a general statement of this part with no indentation

Reply:
If that's the general statement, then I'll suggest to add some basic issue reporting tips on that, e.g. with full logs, full context etc.

Reply:
if go by what you said, then i think it should be moved to the end of the chapter